### ISSUE TITLE
Is the 'inpath' parameter of cmor.cmor_setup used?

### ISSUE THREAD

**Issue by jypeter** [PCMDI/cmor#21]
I'm afraid **there might be an error in the documentation**

I have specified the location of my CMOR tables using *cmor.cmor_setup(inpath='absolute_path_to_the_tables', ...)*, according to what the documentation says on page 12:
```
[inpath] = a character string specifying the path to the directory where the needed
MIP-specific tables reside.
```
But I got an error when executing *cmor.load_table('CMIP5_Amon')*
```
C Traceback:
In function: cmor_load_table
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!                                         !
! Error: Could not find table: CMIP5_Amon !
!                                         !
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

I  have then checked more closely the documentation of *cmor.load_table*  (page 20) and it **does not mention using the path specified by cmor.cmor_setup**:
```
CMOR will look for the table first followig the path
as specified by the “table” argument passed to this function. If it doesn’t find a file
there it will prepend the outpath defined in calling cmor_dataset. If it still doesn’t
find it, it will use the “prefix” where the library CMOR is to be installed (from configure time)
followed by share (e.g /usr/local/cmor/share). If it stills fails an
error will be raised.
```

**Comment by dnadeau4** [comment]
jypeter,

This does not seem to be a problem  Here is the code I use to load the table.
```python
        # -----------------------------------
        # Take care of cmor initialization.
        # Tables are located under ${BIN}/Tables/   
        # -----------------------------------
        cmor.setup(inpath="Tables",
                   netcdf_file_action = cmor.CMOR_REPLACE)
      
        cmor.dataset(experiment_id = rc['experiment_id'], 
                     institution   = rc['institution'  ],
                     calendar      = rc['calendar'     ],
                     institute_id  = rc['institute_id' ],
                     model_id      = rc['model_id'     ],
                     source        = rc['source'       ],
                     contact       = rc['contact'      ],
                     references    = rc['references'   ])
      

        cmor.load_table("CMIP_Amon")
```

Do you still have this issue?

**Comment by dnadeau4** [comment]
This have been fixed in cmor 3.1
